aztk/clusterlib.py

```diff
@@ -359,7 +366,7 @@ def pretty_dedicated_node_count(self, cluster: ClusterModel)-> str:
             return '{}'.format(cluster.dedicated_nodes)
 
     def pretty_low_pri_node_count(self, cluster: ClusterModel)-> str:
-        if (cluster.pool.allocation_state is batch_models.AllocationState.resizing\
+        if (cluster.pool.allocation_state is batch_models.AllocationState.resizing
```

Comment:
removed \ only on this line?

Reply:
I don't think it's necessary. It is done the same way in pretty_dedicated_node_count() just above this.

Reply:
i mean there is one next line too that is not needed